l10n_br_base/models/__init__.py

```diff
@@ -1,10 +1,60 @@
 # -*- coding: utf-8 -*-
+#
 # Copyright (C) 2015  Renato Lima - Akretion
+# Copyright 2016 Taŭga Tecnologia
+#   Aristides Caldeira <aristides.caldeira@tauga.com.br>
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+#
 
-from . import res_bank
-from . import res_partner
-from . import res_country
-from . import res_company
-from . import l10n_br_base
-from . import res_config
+from __future__ import division, print_function, unicode_literals
+
+#
+# Sincronização de casas decimais
+#
+from . import inherited_decimal_precision
+from . import inherited_res_currency
+
+#
+# Templates de email
+#
+from . import inherited_mail_template
+
+#
+# Model Base
+#
+from . import sped_base
+
+#
+# Tabelas Geográficas
+#
+from . import sped_pais
+from . import sped_estado
+from . import sped_municipio
+
+#
+# Cadastros básicos
+#
+from . import sped_participante
+from . import inherited_res_company
+from . import inherited_res_partner
+from . import sped_empresa
+
+#
+# Parcelamentos e pagamentos; bancos e contas bancárias
+#
+from . import copied_account_payment_term
+from . import copied_account_payment_term_line
+from . import inherited_account_payment_term
```

Comment:
this code is dependent on the account module, it should go in the l10n_br_account module or even more terminal modules instead